packages/app/providers/wagmi-config.ts

```diff
@@ -4,14 +4,14 @@ import { gnosis, mainnet } from "wagmi/chains";
 import { jsonRpcProvider } from "wagmi/providers/jsonRpc";
 import { ChainId } from "@stackly/sdk";
 import { SafeConnector } from "wagmi/connectors/safe";
-import { RPC_GNOSIS, RPC_MAINNET } from "@/constants";
+import { RPC_LIST } from "@/constants";
 
 const chainJsonRpc: Record<number, { http: string }> = {
   [ChainId.GNOSIS]: {
-    http: RPC_GNOSIS,
+    http: RPC_LIST[100],
   },
   [ChainId.ETHEREUM]: {
-    http: RPC_MAINNET,
+    http: RPC_LIST[1],
```

Comment:
WeE must use `chainId` here

Reply:
I don't fully understand why or how we should use chainId here if we're setting up the RPC URL for each chain.

Could you please elaborate a bit? :c

Reply:
I think it's just in case chain id changes for GNOSIS or ETHEREUM.  Also it's more explicit in the code if you don't know what does the values 1 or 100 mean.

By using `RPC_LIST[ChainId.GNOSIS]` we don't have to care about that.

Reply:
Instead of using 1 let's user `ChainId.ETHEREUM`, this way we use the constant. Same for Gnosis.

Reply:
Ohhh you mean something like `ChainId.Gnosis` for example? 
If that's so, I get it now, fully agree